client/my-sites/comments/comment-list/index.jsx

```diff
@@ -189,6 +203,7 @@ export class CommentList extends Component {
 	setCommentStatus = ( comment, status, options = { isUndo: false, doPersist: false, showNotice: true } ) => {
 		const { commentId, postId, isLiked } = comment;
 		const { isUndo, doPersist, showNotice } = options;
+		const alsoUnlike = isLiked && ( 'approved' !== status );
```

Comment:
I'd still keep the also and make it `alsoUnlikeComment` (and then `also_unlike_comment` in the stats payload). We've used this `also` naming on many other places for side effects.